container/go/pkg/utils/utils.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/bazelbuild/rules_docker/container/go/pkg/compat"
+	"github.com/bazelbuild/rules_docker/container/go/pkg/oci"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
+	"github.com/pkg/errors"
+)
+
+// ArrayFlags are defined for flags that may have multiple values.
```

Comment:
for `string` flags that may have ...

Also name the struct with `string` in there somewhere.